docs/getting-started/deployment.md

```diff
@@ -0,0 +1,275 @@
+
+plutil -insert containerEngine -dictionary "$DEFAULTS"
+plutil -insert containerEngine.name -string moby "$DEFAULTS"
+
+plutil -insert kubernetes -dictionary "$DEFAULTS"
+plutil -insert kubernetes.enabled -bool false "$DEFAULTS"
+```
+
+#### Lock allowed images list to only allow "busybox" and "nginx"
+
+```
+LOCKED=~/Library/Preferences/io.rancherdesktop.profile.locked.plist
+plutil -create xml1 "$LOCKED"
+
+plutil -insert containerEngine -dictionary "$LOCKED"
+plutil -insert containerEngine.enabled -bool true "$LOCKED"
+
+plutil -insert containerEngine.patterns -array "$LOCKED"
+plutil -insert containerEngine.patterns -string busybox -append "$LOCKED"
+plutil -insert containerEngine.patterns -string nginx -append "$LOCKED"
+```
+
+#### Verify the plist files
+
+```xml title="~/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>containerEngine</key>
+	<dict>
+		<key>name</key>
+		<string>moby</string>
+	</dict>
+	<key>kubernetes</key>
+	<dict>
+		<key>enabled</key>
+		<false/>
+	</dict>
+</dict>
+</plist>
+```
+
+```xml title="~/Library/Preferences/io.rancherdesktop.profile.locked.plist"
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>containerEngine</key>
+	<dict>
+		<key>enabled</key>
+		<true/>
+		<key>patterns</key>
+		<array>
+			<string>busybox</string>
+			<string>nginx</string>
+		</array>
+	</dict>
+</dict>
+</plist>
+```
+
+</TabItem>
+<TabItem value="Linux">
+
+On Linux the deployment profiles are stored in JSON format.
+
+The locations for the profiles are:
+
+```
+/etc/rancher-desktop/defaults.json
+/etc/rancher-desktop/locked.json
+~/.config/rancher-desktop.defaults.json
+~/.config/rancher-desktop.locked.json
+```
+
+#### Convert all current settings into a deployment profile
+
+Since deployment profiles are stored in JSON format, the simplest way to create them is by saving the current application settings to the profile location, and then fine-tuning the profile with a text editor.
+
+```
+rdctl list-settings > ~/.config/rancher-desktop.defaults.json
+```
+
+#### By default use the "moby" container engine and disable Kubernetes
+
+```json title="~/.config/rancher-desktop.defaults.json"
+{
+  "containerEngine": {
+    "name": "moby"
+  },
+  "kubernetes": {
+    "enabled": true
+  }
+}
+```
+
+#### Lock allowed images list to only allow "busybox" and "nginx"
+
+```json title="~/.config/rancher-desktop.locked.json"
+{
+  "containerEngine": {
+    "allowedImages": {
+      "enabled": true,
+      "patterns": ["busybox","nginx"]
+    }
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Known Issues and Limitations
+
+* The `containerEngine.allowedImages` settings are currently the only ones that can be locked.
```

Comment:
This is not quite true.  `containerEngine.allowedImages.enabled` and `...patterns` are the only settings where if they're locked, it shows up in the UI. The UI disables them, shows a lock icon with a tooltip.

The UI gives the impression that other locked fields can be modified, but the user will get an error message if they try to apply the changes.

`rdctl set` will give an error if the command attempts to change a locked field's value (but it's not an error to try to change a locked field to the same value).

`rdctl start` and running the underlying app with a command-line option that would change a locked field's value should result in a failure to start the app.

Reply:
Ok, I will need to test this. This is not what we agreed on during the feature planning meeting, but we can leave it as-is and change the "Known issue".

Reply:
You're right -- I just looked at the code, and we only allow those `containerEngine.allowedImages` fields to be locked.

Reply:
I guess all my testing with the command-line and `rdctl set` fields were on those settings.